swagger.yaml

```diff
@@ -35,6 +35,22 @@ paths:
           description: OSUUID of a person. Public identifier given in Directory API
           type: number
           format: integer
+        - in: query
+          name: firstName
+          required: false
+          description: First name of a person, for fuzzy name searching.
+          type: string
+        - in: query
+          name: lastName
+          required: false
+          description: Last name of a person, for fuzzy name searching.
+          type: string
+        - in: query
+          name: searchOldVersions
+          required: false
+          default: false
+          description: If true, searching by name or OSU ID will also search old names and OSU ID's associated with the person.
+          type: boolean
```

Comment:
Another thought, do you think it will be valuable if we can provide the feature that allows user search by old name/OSU ID?